website/docs/getting-started/usage.md

```diff
@@ -196,10 +196,10 @@ To reduce this risk, ORT filters out certain environment variables when it runs
 This filter mechanism can be configured via the following properties in the
 [ORT configuration file](https://github.com/oss-review-toolkit/ort/blob/main/model/src/main/resources/reference.yml):
 
-| Property | Description |
-|----------|-------------|
-| deniedProcessEnvironmentVariablesSubstrings | A list of substrings that identify variables containing sensitive information. All variables that contain at least one of these strings (ignoring case) are not propagated to child processes. The default for this property contains strings like "PASS", "PWD", or "TOKEN", which are typically used to reference credentials. |
-| allowedProcessEnvironmentVariableNames | This is a list of variable names that are explicitly allowed to be passed to child processes - even if they contain a substring listed in `deniedProcessEnvironmentVariablesSubstrings`. Via this property variables required by external tools, e.g. credentials for repositories needed by package managers, can be passed through. Here, entries must match variables names exactly and case-sensitively. |
+| Property                                    | Description                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
This was flagged by a code inspection hint. This commit silences it.

Reply:
Please provide the details in the commit message then (which tool was used, which is the rule that was violated). Esp. as none of our [automated Markdown checks](https://github.com/oss-review-toolkit/ort/blob/cc0865e310470706aef0b93f9b62d58052639d68/.github/workflows/static-analysis.yml#L61-L81) seem to cover this, we need to find an agreement on what tool recommendations to follow, and which not.

Reply:
@sschuberth commit message has been updated, can I keep it or shall I drop the commit?

Reply:
I'd like to hear @mnonnenmacher's opinion on that as he has recently addressed a bunch of Markdown style issues. Personally, I have a slight tendency to drop it, as we'd otherwise need to do the same change for all Markdown tables for consistency, and I prefer to be able to rely on our automated Markdown checks (but maybe there's a rule that we could enable, if this change is desired).

Reply:
I've split this out into a separate PR, so let's continue discussion there: https://github.com/oss-review-toolkit/ort/pull/7648.